configure.ac

```diff
@@ -1444,13 +1444,21 @@ if test -n "$LIBAIO"; then
 fi
 
 BUILD_LIBURING=no
+AC_ARG_ENABLE([linux-io_uring],
+              AC_HELP_STRING([--disable-linux-io_uring],
+                             [Disable io-uring in POSIX xlator.]))
 AC_CHECK_HEADERS([liburing.h])
 AC_CHECK_LIB([uring],[io_uring_queue_init],[LIBURING="-luring"])
 
-if test -n "$LIBURING"; then
-   AC_DEFINE(HAVE_LIBURING, 1, [io-uring based POSIX enabled])
-   BUILD_LIBURING=yes
+if test "x$enable_linux-io_uring" != "xno"; then
```

Comment:
```suggestion
if test "x$enable_linux_io_uring" != "xno"; then
```

Reply:
Thanks a lot @xhernandez , this works! Is it documented in automake docs that hyphens must be converted to underscore?

Reply:
I haven't found it explicitly, but I've found some references to special characters being translated to `_`, and `-` is not valid in a variable name in shell scripts.